heron/lib/widgets/BookmarksPanel.js

```diff
@@ -645,21 +698,24 @@ Heron.widgets.BookmarksPanel = Ext.extend(Heron.widgets.HTMLPanel, {
 	},
 
 	isValidBookmark: function (context) {
+        var excludeLayers = ['Editor', 'OpenLayers.Handler.Polygon', 'OpenLayers.Handler.RegularPolygon', 'OpenLayers.Handler.Path', 'OpenLayers.Handler.Point']; // Layer-names to be excluded from validation, edit-Layers

```

Comment:
I didn't realize that the Editing Layer is not always named 'Editor'. I will change the implementation.